MSH/msh_mesh.cpp

```diff
@@ -1506,7 +1506,7 @@ void CFEMesh::GetNODOnPLY(const GEOLIB::Polyline* const ply,
 	}
 
 	// compute nodes (and supporting points) along polyline
-	double search_radius(this->getMinEdgeLength()); // getSearchLength());
+	double search_radius(this->getMinEdgeLength()/2); // getSearchLength());
```

Comment:
Find another solution by using $EPSILON in the polyline definition, for example
```
$EPSILON
  1.0e-4  
``` 

The reason is that ``CFEMesh::_min_edge_length``, which is return by  ``CFEMesh::getMinEdgeLength()``, is reset to
`` m_polyline->epsilon``.

In the code, ``m_msh->setMinEdgeLength(m_polyline->epsilon);`` is called before ``m_msh->GetNODOnPLY``.